src/main/resources/application-autotest.yml

```diff
@@ -6,14 +6,21 @@ logging:
       gov.usds.case_issues: DEBUG
       org.springframework.security.access: DEBUG
 spring:
+  profiles:
+    include: autotest-local
```

Comment:
This is a little bit of a cheeky approach, but it turns out to have been useful almost immediately.